src/components/FormModal/Checkbox.js

```diff
@@ -1,14 +1,14 @@
 import "./Checkbox.css";
-import { FiMoon, FiSun } from "react-icons/fi";
+
 import { getClassForWeekdayCheckboxAlignment } from "../../utility/getClassesForSizingAndPositioning";
 import PropTypes from "prop-types";
+import TimeOfDayCheckbox from "./TimeOfDayCheckbox";
 
 Checkbox.propTypes = {
   name: PropTypes.string.isRequired,
   isChecked: PropTypes.bool.isRequired,
   onDayClicked: PropTypes.func.isRequired,
-  onMorningClicked: PropTypes.func.isRequired,
-  onEveningClicked: PropTypes.func.isRequired,
+  onTimeOfDayClicked: PropTypes.func.isRequired,
```

Comment:
vague?

Reply:
vague on purpose :D can be either morning or evening